test/unit/test_tricrypto.py

```diff
@@ -435,3 +435,38 @@ def test_dydxfee(vyper_tricrypto):
         dx *= precisions[i]
         dy *= precisions[j]
         assert abs(dydx - dy / dx) / (dy / dx) < 1e-4
+
+
+@given(positive_balance, positive_balance, positive_balance)
+@settings(
+    suppress_health_check=[HealthCheck.function_scoped_fixture],
+    max_examples=5,
+    deadline=None,
+    phases=[Phase.reuse, Phase.generate, Phase.target],
```

Comment:
I would just use the default phases unless you find it really causes an issue here.